test/analysis-tests.ts

```diff
@@ -35,12 +37,11 @@ describe('LLVM-mca tool definition', () => {
 
     before(() => {
         ce = makeCompilationEnvironment({languages});
-        const info = {
-            exe: null,
+        const info: Partial<CompilerInfo> = {
             remote: true,
             lang: languages.analysis.id,
         };
-        a = new LLVMmcaTool(info, ce);
+        a = new LLVMmcaTool(info as CompilerInfo, ce);
```

Comment:
See comment regarding test

Reply:
I can change this to use makeFakeCompilerInfo, to stay consistent with other code.